iree/compiler/Dialect/Flow/Transforms/HLOToHLOPreprocessing.cpp

```diff
@@ -114,6 +114,16 @@ class ExtractConvOpPaddingAttributes : public OpRewritePattern<mhlo::ConvOp> {
     if (!hasPadding(op)) return failure();
     auto inputType = op.lhs().getType().cast<ShapedType>();
     int rank = inputType.getRank();
+
+    // We can't extract padding if the left hand side has dilation.
+    if (op.lhs_dilation().hasValue()) {
```

Comment:
This means we can't codegen convtranspoe: Lets add a TODO here, we have to figure out either:
1) Have codegen lowering for conv + padding
2) Extract correct padding argument in the transpose_conv case

Reply:
Updated.

Reply:
(I should be able to handle this better in the follow up)